src/disable-amp.js

```diff
@@ -10,11 +10,9 @@ function observeDomChanges(callback) {
 /**
  * Redirects "/amp/" website to a stripped version
  */
-function avoidPersistance() {
-    const url = document.URL;
-    if (url.includes("/amp/")) {
-      document.location.replace(url.substring(0, url.indexOf("amp/")) + "?nonamp=1");
-    }
+const persistentURL = document.URL;
+if (persistentURL.includes("/amp/")) {
+    document.location.replace(persistentURL.substring(0, persistentURL.indexOf("amp/")) + "?nonamp=1");
```

Comment:
unfortunately, it's not a universal solution. it will brake a lot of websites, for example try https://4pda.ru/amp/2020/03/11/369080/

Reply:
Maybe I could change current If condition with an "includes" on last 5 letters of URL.

Reply:
Fixed https://github.com/AdguardTeam/DisableAMP/pull/19/commits/3de56f514105269510185e01252e8dbb9c0b53bf